AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/MainActivity.java

```diff
@@ -67,15 +79,77 @@ public void onSdkInitialized(final AppLovinSdkConfiguration config)
         } );
     }
 
+    private void checkSdkKey()
+    {
+        final String sdkKey = AppLovinSdk.getInstance( getApplicationContext() ).getSdkKey();
+        if ( "YOUR_SDK_KEY".equalsIgnoreCase( sdkKey ) )
+        {
+            new AlertDialog.Builder( this )
+                    .setTitle( "ERROR" )
+                    .setMessage( "Please update your sdk key in the manifest file." )
+                    .setCancelable( false )
+                    .setNeutralButton( "OK", null )
+                    .show();
+        }
+    }
+
+    private Intent getContactIntent()
+    {
+        Intent intent = new Intent( Intent.ACTION_SENDTO );
+        intent.setType( "text/plain" );
+        intent.setData( Uri.parse( "mailto:" + "support@applovin.com" ) );
```

Comment:
From [Android's documentation](https://developer.android.com/guide/components/intents-common#Email), it looks like you would have to set the email using the [`Intent.EXTRA_EMAIL`](https://developer.android.com/reference/android/content/Intent#EXTRA_EMAIL).

Reply:
Updated:

intent.putExtra(Intent.EXTRA_EMAIL, "support@applovin.com");